README.md

```diff
@@ -9,6 +9,9 @@ Intended for internal use at Penn Labs to speed up queries to Open Data.
 Setup
 -----
 * Install [redis](https://redis.io/)
+* Install mysql
+    * Mac OS X: `brew install mysql` and [these instructions](https://solitum.net/openssl-os-x-el-capitan-and-brew/)
+    * Debian/Ubuntu: `apt-get install libmysqlclient-dev`
```

Comment:
I think the setup may be getting complicated enough to have a Dockerfile. Open an issue if you think it would be helpful to your workflow.